Comanche055/TVCROLLDAP.agc

```diff
@@ -600,18 +604,16 @@ MINLIM		DEC	.00277778 B+4	# RATELIM,MIN (1DEG/SEC), SC.AT B-4 REV/SC
 
 MAXLIM		DEC	.01388889 B+4	# RATELIM,MAX (5DEG/SEC), SC.AT B-4 REV/SC
 
-# The following two were B+4.---RSB 2009.
-TMINFIRE	DEC	1.5 B-10	# 15 MS. (14MIN), SC.AT 16 BITS/CS
+TMINFIRE	DEC	1.5 B-10	# 15 MS (14MIN), SC.AT 16 BITS/CS
```

Comment:
Should the code be changed to `B+4` instead of `B-10`?
(see the original transcript and the removed comment in line 603)

Reply:
I think this was some kind of patch that was done to make it work on the simulator, so I say yes, change it back to what the GAP printouts originally said.

Reply:
Will do. 👨‍🚀